### PR TITLE
[KIALI-869] Multiple Namespaces Ansible Playbook

### PR DESCRIPTION
**Pull request by gbaufake**
This playbook will allow deploying a kiali test mesh will allow a flexible mesh on flexible number of namespaces (eg: it can be 2 or 100)

Best Regards,
Guilherme Baufaker Rêgo

### CODE REVIEW

File: test-service/deploy/ansible/README.adoc [hunk 1]
```diff
@@ -51,3 +51,22 @@ image::images/kiali-test-depth-sink.png[]
...
+----
+which
+
+- `number_of_services` is the number of services will be deployed per namespace
+
+- `number_of_versions` is the number of versions each service will have per namespace
+
+- `meshes` specifies which mesh will be installed.
+
+- `number_max_of_namespaces` is the number max of namespaces. The playbook will randomize the choice of namespace for each service (eg: in 2 namespaces of max with 4 services, we could have scenario 1 with: 3 services at namespace-1 and 1 service at namespace-2 or we could have scenario 2 with 2 services on each namespace).
```
Comment (jmazzitelli) [inline test-service/deploy/ansible/README.adoc]: I think this "random" aspect is not going to be useful. This should be more repeatable and more fixed. Think about why we are using these test meshes. We are building these test meshes to run tests and make sure things work correctly. But suppose we see a bug in Kiali that was exposed by this multi-namespace test mesh. We write a bug report and tell the developers about it. The developers are immediately going to ask "how can I replicate this bug?" You can tell them "run the multi-namespace test mesh" but if it is all random, the developer is not guaranteed to get the same test mesh that YOU got when you saw the bug. So it is possible the developer might not be able to replicate the bug since the test mesh isn't going to be guaranteed to be the same.

We should have this more repeatable - I think we should take out the randomness because otherwise won't it be hard to replicate test results?

Comment (cfcosta) [reply test-service/deploy/ansible/README.adoc]: I think both have a case to exist. The random one is like a fuzzing, we can discover some edge cases using it, but the predictable one (that should be default) is to show the expected behaviour.

Maybe we should move the random one to a feature flag?

**Comment by cfcosta** [comment]
One question: what happens if the number of namespaces is bigger than the number of services/versions?

**Comment by gbaufake** [comment]
I can create a check not to allow this scenario to happen.

if number_of_namespaces > number_of_services ... then fail.

**Comment by cfcosta** [comment]
@gbaufake that'd be good.
